src/v/v8_engine/tests/instance_test.cc

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2021 Vectorized, Inc.
+ *
+ * Licensed as a Redpanda Enterprise file under the Redpanda Community
+ * License (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * https://github.com/vectorizedio/redpanda/blob/master/licenses/rcl.md
+ */
+
+#include "seastarx.h"
+#include "v8_engine/environment.h"
+#include "v8_engine/instance.h"
+
+#include <seastar/testing/thread_test_case.hh>
+
+#include <boost/algorithm/string/case_conv.hpp>
+#include <boost/test/unit_test.hpp>
+
+#include <cstdint>
+
+const std::string_view to_upper_case_js = "function to_upper(obj) {\
```

Comment:
nit: why not c++ raw strings, something like 
```cpp
auto js_string = R"js(function to_upper(obj) {
  ...
})js";
```

Reply:
I didn't even know that one can have the multiline strings this way in C++

Reply:
I delete this
And use js files for testing